### DISCUSSION TITLE
[Feature Request] Album art in playlist

### DISCUSSION THREAD

**Discussion by snatrio** [strawberrymusicplayer/strawberry#961] [Ideas]
Hi, it would be nice if we can add an Album Art field in the playlist columns (and resizable rows too), what do you guys think?

Cheers.

**Comment by bhankas** [comment]
There is limited utility IMO. Currently playing Album Art is already displayed on bottom left so that is covered. Extra purpose of having album art in rows would be to identify and differentiate tracks better, but then we run into few things:

1. I like compact view. At that size, any album art is illegible, even on HDPi displays.
2. Increasing row height means lots of whitespace
3. When albums are played one after another, multiple tracks would have same album art in row, making their differentiation purpose moot.

All said, I'm not sure adding this feature will provide sufficient value for the maintenance overhead.

**Comment by RAlexiev** [comment]
When we load to a playlist multiple albums, we should be able to group them visually somehow for more pleasant view and ease of navigation. Also there is no problem with keeping the list compact wile grouping songs like that. It could be a large album cover on the left with all album songs listed on the right for each album. Other players are doing that, and at least for me this is the one thing I really miss in Strawberry.

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
